### PR TITLE
[14.0][IMP] account_move_line_stock_info: Add init hook

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
We want to make sure that we move the stock_move_id from the account.move
to the account.move.line

@NuriaXForgeFlow @AaronHForgeFlow

### CODE REVIEW

File: account_move_line_stock_info/hooks.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+def post_init_hook(cr, registry):
+
+    """ INIT stock references in acount move line """
+    # FOR stock moves
+    cr.execute(
+        """
+        UPDATE account_move_line aml SET stock_move_id = am.stock_move_id
+        FROM account_move am
+        WHERE am.id = aml.move_id
+        AND am.stock_move_id IS NOT NULL;
```
Comment (AaronHForgeFlow) [inline account_move_line_stock_info/hooks.py:14]: Maybe also:
```suggestion
        AND am.stock_move_id IS NOT NULL AND aml.stock_move_id IS NULL;
```

Comment (MiquelRForgeFlow) [reply account_move_line_stock_info/hooks.py:14]:
```suggestion
        AND am.stock_move_id IS NOT NULL AND aml.stock_move_id IS NULL;
```
hehe

Comment (AaronHForgeFlow) [reply account_move_line_stock_info/hooks.py:14]: Opps yes!

**Comment by flotho** [comment]
Hi @JordiBForgeFlow , easy comments to fix from your colleagues.

File: account_move_line_stock_info/hooks.py (hunk 1 above)
Comment (JordiBForgeFlow) [reply account_move_line_stock_info/hooks.py:14]: I don't agree with this. We want to make sure that we overwrite any previous value.

Comment (flotho) [reply account_move_line_stock_info/hooks.py:14]: > I don't agree with this. We want to make sure that we overwrite any previous value.

thanks for this explanation.
I'm not experienced enough on this part to approve or request a change, I let you as expert to "fight"

**Comment by AaronHForgeFlow** [comment]
Rebased and fixed conflicts.

> We want to make sure that we overwrite any previous value.

I don't like to overwrite previous values, but in the real world that column is empty when installing the module, so it is ok to me.

**Review by AaronHForgeFlow — APPROVED** [review]

**Review by flotho — APPROVED** [review]

**Comment by rousseldenis** [comment]
@JordiBForgeFlow Ready ?

**Pull request merged**